iotileemulate/iotile/emulate/internal/emulation_loop.py

```diff
@@ -359,6 +359,16 @@ def add_task(self, tile_address, coroutine):
 
         self._loop.call_soon_threadsafe(self._add_task, tile_address, coroutine)
 
+    def is_tile_busy(self, address):
+        """
+        Check that tile has no pending rpcs
```

Comment:
Please match the surrounding CoreTools docstring style which is:

- Summary line on same line a `"""` character
- One blank line
- any associated long summary
- Args Stanza
    - Each arg should be listed as `arg_name (arg_type): arg_description`
- One blank line
- Returns Stanza

Reply:
Thank you for explanation, I missed some points